src/main/java/org/testng/internal/Parameters.java

```diff
@@ -102,10 +102,14 @@
 
     mapping.put(BeforeMethod.class.getSimpleName(), beforeAfterMethod);
     mapping.put(AfterMethod.class.getSimpleName(), beforeAfterMethod);
-    mapping.put(Test.class.getSimpleName(), Arrays.asList(new Class<?>[] {ITestContext.class}));
+    mapping.put(Test.class.getSimpleName(), newList(ITestContext.class));
 
   }
 
+  private static List<Class<?>> newList(Class<?>...elements) {
```

Comment:
It is totally debatable but I think we don't need a specific method to do a so simple thing which takes only 1 line.